sdk/servicebus/Microsoft.Azure.ServiceBus/src/Primitives/TokenProvider.cs

```diff
@@ -79,7 +79,7 @@ public static TokenProvider CreateSharedAccessSignatureTokenProvider(string keyN
         /// <returns>The <see cref="Microsoft.ServiceBus.TokenProvider" /> for returning Json web token.</returns>
         public static TokenProvider CreateAzureActiveDirectoryTokenProvider(
             AzureActiveDirectoryTokenProvider.AuthenticationCallback authCallback,
-            string authority = AzureActiveDirectoryTokenProvider.CommonAuthority,
+            string authority,
```

Comment:
Should we consider validating the `authority` as not null or empty? Seems better to catch that up front then failing when trying to acquire a token later.